lib/src/replaced_element.dart

```diff
@@ -339,6 +425,16 @@ ReplacedElement parseReplacedElement(
       return RubyElement(
         element: element,
       );
+    case "math":
+      return MathElement(
+        element: element,
+        texStr: r'',
```

Comment:
nit: wouldn't `null` make more sense than an empty string?

Reply:
I agree, I changed it. The latest commit also has some extra changes for styling, if you'd like to review that.